src/ShopCurrencyBundle/views/TCMSFields/TPkgShopCurrency_CMSFieldPrice/postload.view.php

```diff
@@ -30,6 +30,7 @@
                 $this->sqlData[$currencyTypeFieldNameSql] = $oActiveCurrency->id;
             } else {
                 $this-><?= $sFieldName; ?> = $this->sqlData['<?= $sFieldDatabaseName; ?>'];
+                $this-><?= $sFieldName; ?>Formated = $oActiveCurrency->GetFormattedCurrency($this->sqlData['<?= $sFieldDatabaseName; ?>']);
```

Comment:
These two fields / assignments could be made outside of the if-else then.

Reply:
Field assignments moved outside the if.